### ISSUE TITLE
Vertical menu dropdown positioning bug

### ISSUE THREAD

**Issue by HumbleBeck** [Semantic-Org/Semantic-UI-React#3469]
### Steps

- Create vertical menu with dropdown item
- Add property ```simple``` to dropdown component
- Hover dropdown component

### Expected Result
Dropdown menu aligned with menu item

### Actual Result
Dropdown menu positioned under menu item

### Version
semantic-ui-react - 0.84.0
semantic-ui 2.3.3(codesandbox version), also tested with 2.4.2

### Testcase
https://codesandbox.io/s/lxx1mm7907

The problem in 
```
.ui.simple.active.dropdown>.menu, .ui.simple.dropdown:hover>.menu {
...
top: 100%!important;
...
}
```
since it overrides 
```
.ui.vertical.menu .dropdown.item:not(.upward) .menu {
    top: 0;
}
```
which doesn't has !important

**Comment by lubber-de** [comment]
We fixed this in Fomantic-UI 2.7.2 by https://github.com/fomantic/Fomantic-UI/pull/339

**Comment by layershifter** [comment]
@HumbleBeck thanks for the report, but this issue is CSS related. It's already existing in the main repo: https://github.com/Semantic-Org/Semantic-UI/issues/5454
